DEVELOPMENT.md

```diff
@@ -136,3 +136,16 @@ Steps to create a new version of the pipeline CRDs:
 1) Make the required schema changes in `hub`. This will involve changing the conversion code in all older versions.
 1) Register the old schema (e.g. `v1alpha1`) in the controller runtime (see [here](main.go#L56)).
 1) Copy changes into helm version of the CRD to match that generated in `config/crd/bases`.
+
+Steps to install a new CRD version:
```

Comment:
Shouldn't this be in the main documentation? It's for _operators_, not developers.

Reply:
Ok so I ended up overhauling the documentation commit: 2cbb46fa

I didn't think the CRD version upgrade steps fit anywhere in the existing website structure, so I decided to introduce an `Operator Manual` section and add it there. Let me know what you think.